tests/commands/test_commands.py

```diff
@@ -235,22 +223,19 @@ def test_run_start_when_db_already_exists(mocker,
                                           monkeypatch,
                                           run_start_args,
                                           mocked_setup_logging):
-    from bigchaindb import config
     from bigchaindb.commands.bigchaindb import run_start
     from bigchaindb.common.exceptions import DatabaseAlreadyExists
-    mocked_start = mocker.patch('bigchaindb.processes.start')
+    mocked_start = mocker.patch('bigchaindb.start.start')
 
     def mock_run_init():
         raise DatabaseAlreadyExists()
     monkeypatch.setattr('builtins.input', lambda: '\x03')
     monkeypatch.setattr(
         'bigchaindb.commands.bigchaindb._run_init', mock_run_init)
     run_start(run_start_args)
-    mocked_setup_logging.assert_called_once_with(user_log_config=config['log'])
     assert mocked_start.called
```

Comment:
Same question here: why are we no longer asserting the `args`

Reply:
see above